backend/hitas/tests/apis/test_api_apartment_list.py

```diff
@@ -144,6 +153,157 @@ def test__api__apartment__list(api_client: HitasAPIClient):
+                    ),
+                },
+                "building": {
+                    "id": ap1.building.uuid.hex,
+                    "street_address": ap1.building.street_address,
+                    "link": (
+                        f"/api/v1/housing-companies/{hc1.uuid.hex}"
+                        f"/real-estates/{ap1.building.real_estate.uuid.hex}"
+                        f"/buildings/{ap1.building.uuid.hex}"
+                    ),
+                },
+                "apartment": {
+                    "id": ap1.uuid.hex,
+                    "link": f"/api/v1/housing-companies/{hc1.uuid.hex}/apartments/{ap1.uuid.hex}",
+                },
+            },
+            "sell_by_date": str(ap2.completion_date),
+        },
+        {
+            "id": ap2.uuid.hex,
+            "state": ap2.state.value,
+            "type": ap2.apartment_type.value,
+            "surface_area": float(ap2.surface_area),
+            "address": {
+                "street_address": ap2.street_address,
+                "postal_code": hc2.postal_code.value,
+                "city": hc2.postal_code.city,
+                "apartment_number": ap2.apartment_number,
+                "stair": ap2.stair,
+                "floor": ap2.floor,
+            },
+            "links": {
+                "housing_company": {
+                    "id": hc2.uuid.hex,
+                    "display_name": hc2.display_name,
+                    "link": f"/api/v1/housing-companies/{hc2.uuid.hex}",
+                },
+                "real_estate": {
+                    "id": ap2.building.real_estate.uuid.hex,
+                    "link": (
+                        f"/api/v1/housing-companies/{hc2.uuid.hex}/real-estates/{ap2.building.real_estate.uuid.hex}"
+                    ),
+                },
+                "building": {
+                    "id": ap2.building.uuid.hex,
+                    "street_address": ap2.building.street_address,
+                    "link": (
+                        f"/api/v1/housing-companies/{hc2.uuid.hex}"
+                        f"/real-estates/{ap2.building.real_estate.uuid.hex}"
+                        f"/buildings/{ap2.building.uuid.hex}"
+                    ),
+                },
+                "apartment": {
+                    "id": ap2.uuid.hex,
+                    "link": f"/api/v1/housing-companies/{hc2.uuid.hex}/apartments/{ap2.uuid.hex}",
+                },
+            },
+            "completion_date": str(ap2.completion_date),
+            "ownerships": [
+                {
+                    "owner": {
+                        "id": o3.owner.uuid.hex,
+                        "name": o3.owner.name,
+                        "identifier": o3.owner.identifier,
+                        "email": o3.owner.email,
+                    },
+                    "percentage": float(o3.percentage),
+                }
+            ],
+            "sell_by_date": str(ap2.completion_date),
```

Comment:
In this test we are only actually interested in the `sell_by_date` value, so this json could be shortened to only test for that :)
(Would make this a little bit more readable, and require less maintenance when we change the API)

Reply:
Simplified the test.